.github/workflows/test-only.yml

```diff
@@ -30,9 +30,8 @@ jobs:
           distribution: 'zulu'
           java-version: '11'
 
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v4
+
       - name: Check
-        uses: gradle/gradle-build-action@v3
-        with:
-          arguments: check
-          # Disable cache because of corruption
-          cache-disabled: true
```

Comment:
Looks like this was introduced because we had a cache corruption issue with `gradle/gradle-build-action@v2` - see pull request #59. Not sure if we should try again with the default cache policy in the new action and see what happens?

Reply:
Let's try with the default.